playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java

```diff
@@ -352,25 +347,25 @@ public void onCues(List<Cue> cues) {
         }
 
         @Override
-        public void onVideoSizeChanged(int width, int height, int unappliedRotationDegrees, float pixelWidthHeightRatio) {
+        public void onVideoSizeChanged(VideoSize videoSize) {
```

Comment:
@GouravSna  please add null check

Reply:
Marked as NonNull as inside ExoPlayer, it is not null